Sources/Turnstile/Credentials/Credentials.swift

```diff
@@ -13,4 +13,4 @@
  */
 public protocol Credentials { }
 
-public struct Sesssion: Credentials {}
+public struct Session: Credentials {}
```

Comment:
The problem with this change is that this struct is part of the public API and by fixing this typo breaks backwards compatibility.  This is fine but means we need to bump the majorVersion to indicate a breaking change.  However it might be better to create something like this to keep backwards compatibility:
```swift
@available(*, deprecated)
public typealias Sesssion = Session

public struct Session: Credentials {}
```
@edjiang What do you think?